.appveyor.yml

```diff
@@ -5,6 +5,7 @@ environment:
   - TOXENV: "py36"
   - TOXENV: "py37"
   - TOXENV: "py38"
+  - TOXENV: "py39"
```

Comment:
I think AppVeyor doesn't have this, but let's try anyway.

Reply:
ERROR:  py39: InterpreterNotFound: python3.9

Reply:
could we make it optional somehow or we will just count with it until appveyor adds 3.9?

Reply:
I don't know a way to make it optional.